statusHandler/statusMetricsProvider.go

```diff
@@ -95,12 +96,44 @@ func (nd *statusMetrics) Close() {
 }
 
 // StatusMetricsMap will return all metrics in a map
-func (nd *statusMetrics) StatusMetricsMap() (map[string]interface{}, error) {
+func (nd *statusMetrics) StatusMetricsMap() map[string]interface{} {
```

Comment:
This is not used anymore. I assume that you left it in case we need it for later uses where we would like all the metrics in one place

Reply:
Removed